javascript/javascript2/week1/hyf-bay/main.js

```diff
@@ -0,0 +1,25 @@
+console.log("Script loaded");
+
+const products = getAvailableProducts();
+console.log(products);
+
+function renderProducts(productArray) {
+    // your code here
+    const list = document.querySelector("ul");
+    for (const product of productArray) {
+        const li = document.createElement("li");
+        const header = document.createElement("h2");
+        header.innerHTML = product.name;       
+        const price = document.createElement("span");
+        price.innerHTML = "Price: " + product.price;
+        const rating = document.createElement("span");
```

Comment:
Tiny thing: 
I'd call these `priceSpan` or `priceElement` because that's what they are :)